plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -182,9 +183,14 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
                 }
             }
             if (c is WithSupertypes) {
-                c.supertypes.filter { it.key == sourceSet }.map { (s, dris) ->
-                    list(dris, prefix = " : ", sourceSets = setOf(s)) {
-                        link(it.dri.sureClassNames, it.dri, sourceSets = setOf(s))
+                c.supertypes.filter { it.key == sourceSet }.map { (s, typeConstructors) ->
+                    list(typeConstructors, prefix = " : ", sourceSets = setOf(s)) {
+                        link(it.typeConstructor.dri.sureClassNames, it.typeConstructor.dri, sourceSets = setOf(s))
+                        if ( it.typeConstructor.projections.isNotEmpty() ) {
```

Comment:
I think this if is already included in list function

Reply:
Oh, yes, thanks :D